.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,31 +1,33 @@
 ---
 name: "\U0001F41B Bug report"
 about: Create a report to help us fix things
+title: ''
+labels: ''
+assignees: ''
```

Comment:
What does this do?

Reply:
Do you intend to make more changes to this PR, related to auto fill/assigning or other changes, before I review?